v2/dom.go

```diff
@@ -2395,15 +2391,15 @@ func (ctx *CanvasRenderingContext2D) ResetTransform() {
 // Drawing images
```

Comment:
I can't leave the comment on the right line on GitHub, but it seems `CreatePattern` needs the same fix too.